### DISCUSSION TITLE
🚀 Feature request - Access all file field tags within image manipulation tag pair

### DISCUSSION THREAD

**Discussion by jaredchesebro** [ExpressionEngine/ExpressionEngine#4335] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
Would like to be able to use ```{description}``` as well as other file field tags within an on the fly image manipulation tag pair. 

Currently I have to do this to get the description which seems like a waste:
```
<img src="{image:resize width='800' quality='82'}" alt="{image}{description}{/image}">
```

**Describe the solution you'd like**
Would like to have full access to all the file fields within the tag pair:
```
{image:resize width='800' quality='82'}
    <figure>
        <img src="{url}" alt="{description}" title="{title}">
        <caption>{caption}</caption>
    </figure>
{/image:resize}
```

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4475

**Comment by pyrobob83** [comment]
@intoeetive Following on from this discussion, I'm in the same boat as Jared was initially, except that I can't use the `{description}` tag inside the tag pair because I don't use the tag pairs since I need multiple resizes to achieve responsive image sizing via `srcset` and `sizes`.  Consider this File Grid use case:

```
<img class="block w-full"
    loading="lazy"
    alt="{content:file}{description}{/content:file}"
    src="{content:file:resize width='540' height='10000'}"
    srcset="{content:file:resize width='1424' height='10000'} 1424w,
            {content:file:resize width='1080' height='10000'} 1080w,
            {content:file:resize width='712' height='10000'} 712w,
            {content:file:resize width='540' height='10000'} 540w,
            {content:file:resize width='330' height='10000'} 330w"
    sizes="(min-width: 1536px) 712px,
            (min-width: 1280px) calc((100vw - 104px) / 2), 
            (min-width: 1024px) calc((100vw - 80px) / 2),
            (min-width: 768px) calc((100vw - 72px) / 2),
            (min-width: 450px) calc(100vw - 48px),
            calc(100vw - 32px)"
/>
```

Would it be possible to get each of the file field tags available to the single tag too, so in this instance I could use `{content:file:description}`?

**Reply by jaredchesebro** [reply]
I like this idea.
